PocketKit/Sources/SaveToPocketKit/SavedItem/SavedItemViewModel.swift

```diff
@@ -101,6 +106,50 @@ class SavedItemViewModel {
         autodismiss(from: context)
     }
 
+    @MainActor
+    func open(from context: ExtensionContext?) async {
+        guard appSession.currentSession != nil else {
+            autodismiss(from: context)
+
+            return
+        }
+
+        guard let extensionItems = context?.extensionItems else {
+            tracker.track(event: Events.SaveTo.unableToOpen())
+            autodismiss(from: context)
+
+            return
+        }
+
+        guard let url = await parse(extensionItems: extensionItems) else {
+            tracker.track(event: Events.SaveTo.unableToOpen())
+            infoViewModel = .error
+
+            return
+        }
+
+        var components = URLComponents()
+        components.scheme = "pocketSaveTo"
+        components.path = "//itemURL"
+        components.queryItems = [URLQueryItem(name: "url", value: url)]
+
+        guard let context, let deepLinkURL = components.url else {
+            tracker.track(event: Events.SaveTo.unableToOpen())
+            autodismiss(from: context)
+
+            return
+        }
+
+        context.open(url: deepLinkURL, completionHandler: { [weak self] success in
```

Comment:
this isn't opening in the app currently 🤔

Reply:
Not supported in share extensions. :< Per the docs ...

> Each extension point determines whether to support this method, or under which conditions to support this method. In iOS, the Today and iMessage app extension points support this method. An iMessage app extension can use this method only to open its parent app, and only if the parent app is shown on the iOS home screen.